.circleci/push_to_hub.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -ev
+REPO="dddecaf/decaf-metabolica"
+GIT_SHA=$(git rev-parse --short=12 --verify HEAD)
+docker build -f Dockerfile -t $REPO:$CIRCLE_BRANCH .
+docker tag $REPO:$CIRCLE_BRANCH $REPO:$GIT_MASTER_HEAD_SHA
+docker push $REPO:$CIRCLE_BRANCH
+docker push $REPO:$GIT_MASTER_HEAD_SHA
```

Comment:
This script can be deleted as the build/tag/push logic is in circleci config now